### PR TITLE
SapMachine #1348: Fix jtreg test jdk/jfr/api/recording/settings/TestGetConfigurations.java

### LINKED ISSUE #1348: jtreg test  jdk/jfr/api/recording/settings/TestGetConfigurations.java fails
Test fails after two additional JFR configuration files were added with https://github.com/SAP/SapMachine/pull/1336

### PR DESCRIPTION
**Pull request by ansteiner**
Test failed after two additional JFR configuration files were added with https://github.com/SAP/SapMachine/pull/1336

fixes #1348

### CODE REVIEW

**Comment by SapMachine** [comment]
Hello @ansteiner, this pull request fulfills all formal requirements.

File: test/jdk/jdk/jfr/api/recording/settings/TestGetConfigurations.java [hunk 1]
```diff
@@ -48,10 +48,21 @@ public class TestGetConfigurations {
...
+
+    private static final String GC_DETAILS_CONFIG_NAME = "gc_details";
+    private static final String GC_DETAILS_CONFIG_LABEL = "gc_details";
+    private static final String GC_DETAILS_CONFIG_DESCRIPTION = "Configuration for all GC related events. Higher impact caused by heap inspection initiated GCs to get heap statistics. Large recording size.";
+    private static final String GC_DETAILS_CONFIG_PROVIDER = "SAP SE";
+
     public static void main(String[] args) throws Throwable {
         List<Configuration> predefinedConfigs = Configuration.getConfigurations();
         Asserts.assertNotNull(predefinedConfigs, "List of predefined configs is null");
-        Asserts.assertEquals(predefinedConfigs.size(), 2, "Expected exactly two predefined configurations");
```
Comment (RealCLanger) [inline test/jdk/jdk/jfr/api/recording/settings/TestGetConfigurations.java:54]: Please add a //SapMachine comment here as well

File: test/jdk/jdk/jfr/api/recording/settings/TestGetConfigurations.java [hunk 2]
```diff
@@ -64,6 +75,18 @@ public static void main(String[] args) throws Throwable {
         Asserts.assertEquals(profileConfig.getLabel(), PROFILE_CONFIG_LABEL);
         Asserts.assertEquals(profileConfig.getDescription(), PROFILE_CONFIG_DESCRIPTION);
         Asserts.assertEquals(profileConfig.getProvider(), PROFILE_CONFIG_PROVIDER);
+
+        Configuration gcConfig = findConfigByName(predefinedConfigs, GC_CONFIG_NAME);
```
Comment (RealCLanger) [inline test/jdk/jdk/jfr/api/recording/settings/TestGetConfigurations.java:79]: Here too.

**Review by RealCLanger — CHANGES_REQUESTED** [review]

**Comment by SapMachine** [comment]
Hello @ansteiner, this pull request fulfills all formal requirements.

File: test/jdk/jdk/jfr/api/recording/settings/TestGetConfigurations.java (hunk 1 above)
Comment (ansteiner) [reply test/jdk/jdk/jfr/api/recording/settings/TestGetConfigurations.java:54]: done

File: test/jdk/jdk/jfr/api/recording/settings/TestGetConfigurations.java (hunk 2 above)
Comment (ansteiner) [reply test/jdk/jdk/jfr/api/recording/settings/TestGetConfigurations.java:81]: done

**Review by MBaesken — APPROVED** [review]

**Review by RealCLanger — APPROVED** [review]
Looks good now. I guess you verified that it fixes the test 😄

**Pull request merged**